<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PulseOximeterSimulatorClass</class>
 <widget class="QMainWindow" name="PulseOximeterSimulatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PulseOximeterSimulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="MeasureButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSubButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Subject</string>
    </property>
   </widget>
   <widget class="QLabel" name="PRbpmTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PR bpm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PRbpmData">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>65</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="SpOBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>26</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="minimum">
     <number>70</number>
    </property>
    <property name="value">
     <number>95</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpOTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%SpO2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SpOData">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>35</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>94</string>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PulseOximeterSimulator.qrc"/>
  <include location="PulseOximeterSimulator.qrc"/>
 </resources>
 <connections/>
</ui>
